setup.py

```diff
@@ -10,7 +10,7 @@
 ]
 
 dev_dependencies = [
-    "clvm_tools>=0.4.2",
+    "clvm_tools>=0.4.4",
```

Comment:
I believe the tests rely on the most recent version of `clvm_tools` now, since we changed the cost reporting